docs/04-architecture/turbopack.mdx

```diff
@@ -27,3 +27,30 @@ Turbopack in Next.js requires zero-configuration for most users and can be exten
 ## Unsupported features
 
 Turbopack currently only supports `next dev` and does not support `next build`. We are currently working on support for builds as we move closer towards stability.
+
+These features are currently not supported:
+
+- `webpack()` config in `next.config.js`
+  - Turbopack replaces Webpack, this means that webpack configuration is not supported.
+  - For configuring Turbopack [see the documentation](/docs/app/api-reference/next-config-js/turbo).
+  - A subset of Webpack loaders [are supported](/docs/app/api-reference/next-config-js/turbo#webpack-loaders) in Turbopack.
+- `.babelrc`
```

Comment:
```suggestion
-  Babel`(.babelrc`)
```

Unless you think it's fair to assume someone looking for Babel will know to look for `.babelrc`, which is fair.